hackathon/mdakits/projects.md

```diff
@@ -0,0 +1,274 @@
+  that can be either used with a similarity matrix or directly with
+  trajectory data like coordinates.
+  
+* The GROMOS clustering algorithm is widely used in biomolecular
+  simulations [[Daura 1999]](#Daura1999). (See Issue
+  [#2876](https://github.com/MDAnalysis/mdanalysis/issues/2876and has been requested)
+  
+    "To find clusters of structures in a trajectory the RMSD of atom
+    positions between all pairs of structures was determined. For each
+    structure the number of other structures for which the RMSD was
+    0.1 nm or less (backbone, residues 2 ± 6) for structure 1 or 0.08
+    nm or less (backbone, residues 2 ± 5) for structure 2 (neighbor
+    conformations) was calculated. The structure with the highest
+    number of neighbors was taken as the center of a cluster, and
+    formed together with all its neighbors a (first) cluster. The
+    structures of this cluster were thereafter eliminated from the
+    pool of structures. The process was repeated until the pool of
+    structures was empty. In this way, a series of nonoverlapping
+    clusters of structures was obtained."
+
+### Objectives ###
+
+- Create a `ClusterAnalysis` class that allows the user to run any of
+   the [scikit-learn clustering][] algorithms that can work on raw
+   data (such as K-means). Use the `AnalysisBase` framework to write
+   the analysis class (see the [tutorial on writing your own
+   trajectory analysis][UserGuide AnalysisBase].
+- Create an MDAKit that makes `ClusterAnalysis` available.
+- Implement additional clustering methods such as GROMOS clustering
+   [[Daura 1999]](#Daura1999) (described above in more detail).
+
+
+### References ###
+
+1. <a id="Daura1999"/>X Daura, K Gademann, B Jaun, D Seebach, WF van Gunsteren, and AE Mark. Peptide folding: When simulation meets experiment. Angew. Chem. Int. Ed., 38(1-2):236–240, 1999. doi: [10.1002/(SICI)1521-3773(19990115)38:1/2<236::AID-ANIE236>3.0.CO;2-M](https://doi.org/10.1002%2F%28SICI%291521-3773%2819990115%2938%3A1%2F2%3C236%3A%3AAID-ANIE236%3E3.0.CO%3B2-M).
+
+[metric]: https://en.wikipedia.org/wiki/Metric_space#Definition
+[clustering]: https://en.wikipedia.org/wiki/Cluster_analysis
+
+[scikit-learn clustering]: https://scikit-learn.org/stable/modules/clustering.html
+
+[UserGuide AnalysisBase]: https://userguide.mdanalysis.org/stable/examples/analysis/custom_trajectory_analysis.html
+
+
+## Clustered density analysis ##
+
+In addition to identifying different states of a biomolecule by 
+clustering over e.g. RMSD (as described above), clustering can 
+also be used to identify e.g. small molecule/ion binding sites 
+or sidechain configurations by aligning to a ‘fixed’ reference 
+(e.g. protein) and performing clustering using the coordinates 
+of the group of interest. Subsequent density calculations for 
+each cluster can allow for further comparison and visualization.  
+
+### Where to start ###
+- (Independently or in conjunction with the above, create a
```

Comment:
Unless I've misinterpreted anything, seems like there will be an initial overlap in the project and the above with creating an easier clustering interface. Not sure if we want to leave them both (and potentially have two kits with different ClusterAnalysis, move this one to be an optional extension of the above, or remove the cluster itself here and make it just a Kit for generating densities given a series of frame indices?

Reply:
You're right, let's do exactly what you suggest.

Reply:
I rewrote it to "State-based density analysis"